test/integration/request_test.rb

```diff
@@ -37,6 +37,9 @@ def test_request_times
     assert_equal 1, aggregate["rack.request.time"][:count],
       'should track total request time'
 
+    # should calculte p95 value
+    refute_equal aggregate.fetch("rack.request.time", percentile: 95), 0.0
```

Comment:
@nextmat what do you think of this as a way to ensure we don't lose the p95 again in the future? If we've asked for the percentile, this should be a non-zero value.